v2/api-validator/tests/server-tests/pagination-params.test.ts

```diff
@@ -29,6 +31,14 @@ describe('Pagination params tests', () => {
       const querystring = fakeObject(schema.querystring);
       const params = fakeObject(schema.params);
 
+      try {
+        if (params !== undefined) {
+          params.accountId = getCapableAccountId(schema.tags[0] as keyof ApiComponents);
```

Comment:
There might be endpoints without `accountId`. I'd check if there is such a property first, using `hasOwnProperty`. 

Also, if you use `findCapableAccountId`, you'll be able to check the return value and won't need the empty catch.